Tests/Data/Parabolic/T/3D_3BHE_array/pre/3bhes.py

```diff
@@ -0,0 +1,152 @@
+'''
+3 bhes array model
+'''
+from tespy import cmp, con, nwk, hlp
+from tespy import nwkr
+from sub_btes import btes_para as bp
+
+import numpy as np
+import pandas as pd
+import math
+# %% network
+btes = nwk.network(fluids=['water'],
+				 T_unit='K', p_unit='bar', h_unit='kJ / kg',
+				 T_range=[273.25, 373.15], p_range=[1, 100], h_range=[1, 1000])
+
+# components(components sequence important, e.g. bhe name sequence must set with bhe1,2,3...)
+
+# cycle closer (from consumer)
+fc_in = cmp.source('from consumer inflow')
+fc_out = cmp.sink('from consumer outflow')
+
+p1 = cmp.pump('pump')
+
+sp = cmp.splitter('splitter',num_out = 3)
+
+#bhe:
+bhe1 = cmp.heat_exchanger_simple('BHE1')#!name must be same as the user setting in inp_network.py
+bhe2 = cmp.heat_exchanger_simple('BHE2')
+bhe3 = cmp.heat_exchanger_simple('BHE3')
+
+mg = cmp.merge('merge',num_in = 3)
+
+cons = cmp.heat_exchanger_simple('consumer')
+# two btes subsystems (pipes in parallel flow):
+# 10 pipes for outer ring, 5 pipes inner ring
+#bo = bp('btes outer', 4)
+#bi = bp('btes inner', 2)
```

Comment:
There is quite a bit of commented code in the python modules. Is it needed?

Reply:
No, it is no needed. The unnecessary commented codes in python modules will be cleaned.